src/domain/session/room/LightboxViewModel.js

```diff
@@ -92,6 +96,6 @@ export class LightboxViewModel extends ViewModel {
     }
 
     close() {
-        this.platform.history.pushUrl(this.closeUrl);
+        this.history.pushUrl(this.closeUrl);
```

Comment:
Since we're using a custom `History` implementation in https://github.com/matrix-org/matrix-public-archive/pull/12 to get hashes relative to the room, we need to use that custom `History` over the default one from `platform`.

This is the only other place we use `platform.history` outside of [`src/platform/web/main.js#L38`](https://github.com/vector-im/hydrogen-web/blob/8b2299852eea7a2bc0e247de6ccdb2676b3e4665/src/platform/web/main.js#L38)